modules/button/css/lib/button.scss

```diff
@@ -187,11 +187,11 @@ $_wdc-btn-caret-down-icon-hover: svg-load(
     }
 
     &.wdc-btn-active,
-    &:active {
+    &:active:not([disabled]):not(.wdc-btn-disabled) {
```

Comment:
These have to override:
https://github.com/Workday/canvas-kit/blob/4c97e3789716469aae19292d61c5b96bc16c02c4/modules/button/css/lib/button.scss#L178